ngx_pq_module.c

```diff
@@ -914,11 +917,12 @@ static void ngx_pq_finalize_request(ngx_http_request_t *r, ngx_int_t rc) {
     ngx_http_upstream_t *u = r->upstream;
     u->keepalive = !u->headers_in.connection_close;
     u->request_body_sent = 1;
+    ngx_pq_loc_conf_t *plcf = ngx_http_get_module_loc_conf(r, ngx_pq_module);
```

Comment:
I think it would be better to place this line before the actual use of the `plcf` variable.